README.md

```diff
@@ -87,17 +87,16 @@ the following should do:
 ```
 $ brew install alure pkg-config
 $ git clone https://github.com/zevv/bucklespring.git && cd bucklespring
-$ sed -i '' 's/-Wall -Werror/-Wall/' Makefile
 $ make
 $ ./buckle
 ```
 
-Note that you need superuser privileges to create the event tap on Mac OS X.
+Note that you might need superuser privileges to create the event tap on Mac OS X.
```

Comment:
It's been called macOS for a couple years now, with the last version being `XI`/`11` instead of `X`/`10`